src/NuGet.Core/NuGet.Packaging/NuGet.Packaging.csproj

```diff
@@ -48,8 +48,6 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(IsCore)' == 'true' ">
-    <PackageReference Include="System.Dynamic.Runtime" />
-    <!-- TODO - remove this when temporary patching is no longer necessary. See https://github.com/nuget/home/issues/8952 -->
     <PackageReference Include="System.Security.Cryptography.Pkcs" />
     <PackageReference Include="System.Security.Cryptography.Cng" />
```

Comment:
We got a green build 🥳

But when I took a last look at the PR, I noticed the above TODO comment was removed. The comment was in reference to these two crypto packages, so shouldn't be removed as part of this PR.

However, I don't believe that the package references can be removed (for exactly the reason we saw in this PR - it breaks the build). So it's probably ok to remove the comment. But I need to confirm before merging. Otherwise if you put the comment back, I'll merge.

Reply:
Unless there are plans to integrate these two libraries into .NET 5+ proper, their references will be never removed. Since it doesn't seem to be a temporary measure, I don't think such a notice would be necessary.

Reply:
I don't think we'll be able to remove this for now. 
Deleting the comment is correct, so personally I'm in support of taking the change as is.

Reply:
Yes, @heng-liu  also agreed: https://github.com/NuGet/Home/issues/8952#issuecomment-657165603